__tests__/articles.test.js

```diff
@@ -248,3 +249,54 @@ describe("GET /api/articles", () => {
     });
   });
 });
+
+// POST COMMENTS
+describe("POST /api/articles/:article_id/comments", () => {
+  describe("HAPPY PATH", () => {
+    it("returns the posted comment", () => {
+      return request(app)
+        .post("/api/articles/2/comments")
+        .send({
+          author: "icellusedkars",
+          body: "My code is broken.",
+        })
+        .expect(201)
+        .then((res) => {
+          const comment = {
+            comment_id: 19,
+            body: "My code is broken.",
+            article_id: 2,
+            author: "icellusedkars",
+            votes: 0,
+            created_at: expect.any(String),
+          };
+
+          expect(res.body).toMatchObject(comment);
+        });
+    });
+  });
+
+  describe("UNHAPPY PATH", () => {
+    it("404: not found - if article_id doesn't exist", () => {
+      return request(app)
+        .post("/api/articles/57")
+        .expect(404)
```

Comment:
I think this might want to be `/api/articles/57/comments`, otherwise you will be making a request to a different endpoint.